src/ucp/core/ucp_context.c

```diff
@@ -455,6 +455,15 @@ static ucs_config_field_t ucp_context_config_table[] = {
    ucs_offsetof(ucp_context_config_t, reg_nb_mem_types),
    UCS_CONFIG_TYPE_BITMAP(ucs_memory_type_names)},
 
+  {"RMA_OFFLOAD", "try",
+   "Prefer transports capable of remote memory access for RMA and AMO operations.\n"
```

Comment:
IMO calling it "prefer" is inconsistent with the meaning of 'y' value. It is more like "force" (e.g. as in `UCX_DC_MLX5_DCI_FULL_HANDSHAKE`).